File: CompiladoresProyFinal/Ejemplos/app_generada.js
const express = require('express');
const app = express();
app.use(express.json());

app.get('/api/reportes', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes ejecutado con éxito' });
});

app.get('/api/reportes/:id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id ejecutado con éxito' });
});

app.post('/api/reportes', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes ejecutado con éxito' });
});

app.patch('/api/reportes/:id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id ejecutado con éxito' });
});

app.delete('/api/reportes/:id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id ejecutado con éxito' });
});

app.get('/api/reportes/:id/detalles', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/detalles ejecutado con éxito' });
});

app.post('/api/reportes/:id/detalles', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/detalles ejecutado con éxito' });
});

app.delete('/api/reportes/:id/detalles/:detalle_id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/detalles/:detalle_id ejecutado con éxito' });
});

app.get('/api/reportes/:id/exportaciones', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/exportaciones ejecutado con éxito' });
});

app.post('/api/reportes/:id/exportaciones', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/exportaciones ejecutado con éxito' });
});

app.get('/api/reportes/:id/exportaciones/:exportacion_id', (req, res) => {
    res.status(200).json({ message: 'Endpoint /api/reportes/:id/exportaciones/:exportacion_id ejecutado con éxito' });
});

app.listen(3000, () => {
    console.log('Servidor ReportesService corriendo en http://localhost:3000');
});

File: CompiladoresProyFinal/Ejemplos/app_generada.py
from flask import Flask, jsonify, request
app = Flask('ReportesService')

@app.route('/api/reportes', methods=['GET'])
def get__reportes():
    return jsonify({'message': 'Endpoint /api/reportes ejecutado con exito'}), 200

@app.route('/api/reportes/<id>', methods=['GET'])
def get__reportes_id():
    return jsonify({'message': 'Endpoint /api/reportes/<id> ejecutado con exito'}), 200

@app.route('/api/reportes', methods=['POST'])
def post__reportes():
    return jsonify({'message': 'Endpoint /api/reportes ejecutado con exito'}), 200

@app.route('/api/reportes/<id>', methods=['PATCH'])
def patch__reportes_id():
    return jsonify({'message': 'Endpoint /api/reportes/<id> ejecutado con exito'}), 200

@app.route('/api/reportes/<id>', methods=['DELETE'])
def delete__reportes_id():
    return jsonify({'message': 'Endpoint /api/reportes/<id> ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/detalles', methods=['GET'])
def get__reportes_id_detalles():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/detalles ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/detalles', methods=['POST'])
def post__reportes_id_detalles():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/detalles ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/detalles/<detalle_id>', methods=['DELETE'])
def delete__reportes_id_detalles_detalle_id():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/detalles/<detalle_id> ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/exportaciones', methods=['GET'])
def get__reportes_id_exportaciones():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/exportaciones ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/exportaciones', methods=['POST'])
def post__reportes_id_exportaciones():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/exportaciones ejecutado con exito'}), 200

@app.route('/api/reportes/<id>/exportaciones/<exportacion_id>', methods=['GET'])
def get__reportes_id_exportaciones_exportacion_id():
    return jsonify({'message': 'Endpoint /api/reportes/<id>/exportaciones/<exportacion_id> ejecutado con exito'}), 200

if __name__ == '__main__':
    app.run(debug=True, port=5000)